ios/FluentUI/Core/SwiftUI+ViewModifiers.swift

```diff
@@ -61,3 +61,16 @@ extension View {
                               shouldScale: shouldScale))
     }
 }
+
+extension Locale {
```

Comment:
hmmm it is nothing to do with the view modifier. might as well create a new file?

Reply:
Would you recommend creating a new file or renaming this one to just SwiftUI+Modifiers?

Reply:
Personally I vote for a separate file - I'd rather a dozen files with a single method each, versus a single file with a dozen unrelated objects (which is what will probably happen to this file eventually if we make it not View-specific).

A new file should go in the Extensions folder with a name `Locale+Extensions.swift`